src/Sylius/Bundle/ShippingBundle/Behat/ShippingContext.php

```diff
@@ -94,4 +95,12 @@ public function theShippingMethodTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Then The code field should be disable
+     */
+    public function theCodeFiledShouldBeDisabled()
+    {
+        $this->assertSession()->elementAttributeContains('css', '#sylius_shipping_category_code', 'disabled', 'disabled');
```

Comment:
This should be generic, I think we can find this field by label, it will be "Code" on all pages.